frontend/src/tournaments/roundRobin/roundRobinApi.ts

```diff
@@ -0,0 +1,77 @@
+import axios from "axios";
+
+
+export interface TournamentData {
+    info: string;
+    tournamentname: string;
+    pairs: string[][];
+    message: string;
+    desc: string;
+    crosstable: string[][];
+    crosstableString: string;
+    leaderboard: string[];
+    statusCode: number;
+}
+
+export const cohorts = [
+    { label: '0-300', value: 0 },
+    { label: '300-400', value: 300 },
+    { label: '400-500', value: 400 },
+    { label: '500-600', value: 500 },
+    { label: '600-700', value: 600 },
+    { label: '700-800', value: 700 },
+    { label: '800-900', value: 800 },
+    { label: '900-1000', value: 900 },
+    { label: '1000-1100', value: 1000 },
+    { label: '1100-1200', value: 1100 },
+    { label: '1200-1300', value: 1200 },
+    { label: '1300-1400', value: 1300 },
+    { label: '1400-1500', value: 1400 },
+    { label: '1500-1600', value: 1500 },
+    { label: '1600-1700', value: 1600 },
+    { label: '1700-1800', value: 1700 },
+    { label: '1800-1900', value: 1800 },
+    { label: '1900-2000', value: 1900 },
+    { label: '2000-2100', value: 2000 },
+    { label: '2100-2200', value: 2100 },
+    { label: '2200-2300', value: 2200 },
+    { label: '2300-2400', value: 2300 },
+];
+
+const authToken = 'my-token'; 
+const endpoint = 'https://vmqy3k7nj8.execute-api.us-east-1.amazonaws.com';
```

Comment:
Should this be hardcoded in? What about dev/prod URLs?

Reply:
config file!